units/meson.build

```diff
@@ -272,8 +272,7 @@ foreach tuple : in_units
                 file,
                 input : file + '.in',
                 output : file,
-                command : [meson_render_jinja2, config_h, '@INPUT@'],
-                capture : true,
+		command : [meson_render_jinja2, config_h, '@INPUT@', '@OUTPUT@'],
```

Comment:
we use spaces-only for indentation (applies to all meson files)

Reply:
Bah, I forgot about that.

Reply:
Done.